README.md

```diff
@@ -1 +1,41 @@
 # data-integrity-test-suite-assertion
+
+## Table of Contents
+
+- [Background](#background)
+- [Install](#install)
+- [Test](#test)
+- [Usage](#usage)
+- [Implementation](#implementation)
+
+## Background
+This library exports a function `checkDataIntergrity()` that can be called in
```

Comment:
```suggestion
This library exports a function `checkDataIntegrity()` that can be called in
```

Reply:
Fixed in all places [cb10738](https://github.com/digitalbazaar/data-integrity-test-suite-assertion/pull/1/commits/cb107381241336470091a28997cc67bfe603015f)